.semgrep/adapter/package-import.yml

```diff
@@ -0,0 +1,13 @@
+rules:
+  - id: package-import-check
+    message: Importing "$PKG" package is not recommended in adapter code
```

Comment:
Please use stronger wording. These packages are not allowed / banned. It's more than a recommendation.

Reply:
Apologies. PR got merged. Added https://github.com/prebid/prebid-server/pull/2913 to make this change